app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Good job working on this method, but it will be nicer if you implemented a Rails Active Callback `after_save` to call the `update_comments_counter` method every time a new post has been saved to the database, then make the `update_comments_counter` method private. This will update the count every time a new comment is saved.

Reply:
Thank you for the feedback. Done